### DISCUSSION TITLE
"ERROR PCU message size exceeds INT_MAX... exiting

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#993] [Q&A] [answered]
Do you know why this error occurred?
![image](https://github.com/SeisSol/SeisSol/assets/114474894/60bc2429-a482-4010-993c-547a2d2bd8de)
![image](https://github.com/SeisSol/SeisSol/assets/114474894/9dbcf09f-97ec-4deb-ad81-de4a7e1858fa)

**Comment by Thomas-Ulrich** [comment, marked as answer]
I don't know but I would give a try with this branch of pumgen:
https://github.com/SeisSol/PUMGen/tree/davschneller/bypass-apf
